bin/generators/__init__.py

```diff
@@ -0,0 +1,71 @@
+import itertools
+
+import lib
+
+
+def word_generator(length_min=7, length_max=15, perms=False):
+    """
+      Generate the words to be used for bruteforcing
+      > :param length_min: minimum length for the word
+      > :param length_max: max length for the word
+      > :param perms: permutations, True or False
+      > :return: a word
+
+      Example:
+      >>> word_generator()
+      aaaaaa
+      aaaaab
+      aaaaac
+      ...
+    """
+    if not perms:
+        chrs = 'abc'
+        for n in range(length_min, length_max + 1):
+            for xs in itertools.product(chrs, repeat=n):
+                yield ''.join(xs)
+    else:
+        raise NotImplementedError("Permutations are not implemented yet.")
+
+
+def generate_combos(words, verbose=False, **kwargs):
+    """
+      Create combinations of given words
```

Comment:
> _Create combinations of given words_

It seems more like pairs of words but also, it doesn't seem to create all combinations/pairs.

```python
>>> import itertools
>>> for pair in itertools.permutations(["test", "testing", "tests", "spam"], 2):
        print ''.join(pair)
... 
testtesting
testtests
testspam
testingtest
testingtests
testingspam
teststest
teststesting
testsspam
spamtest
spamtesting
spamtests


```